zxlive/graphscene.py

```diff
@@ -52,7 +52,7 @@ def __init__(self) -> None:
         self.setSceneRect(0, 0, 2*OFFSET_X, 2*OFFSET_Y)
         self.setBackgroundBrush(QBrush(QColor(255, 255, 255)))
         self.vertex_map: dict[VT, VItem] = {}
-        self.edge_map: dict[ET, EItem] = {}
+        self.edge_map: dict[tuple[ET, int], EItem] = {}
```

Comment:
Is this correct? You already redefined ET to have an additional int in it.

Reply:
The type of ET has (source, target, edge_type) but this is not enough information to determine which exact edge it is in the drawing. So I give an additional index in the edge_map.